examples/codegen-react-apollo/nhost/config.yaml

```diff
@@ -13,7 +13,7 @@ services:
       postgres_password: postgres
       postgres_user: postgres
   auth:
-    image: nhost/hasura-auth:0.16.1
+    image: nhost/hasura-auth:0.16.2
```

Comment:
Is this change related to the Azure AD introduction?

Reply:
I think I got this when I merged the main in this branch! It seems unrelated and can be removed.